openslides/config/serializers.py

```diff
@@ -0,0 +1,9 @@
+from openslides.utils.rest_api import serializers, VariableField
+
+
+class ConfigSerializer(serializers.Serializer):
```

Comment:
Config REST API changes split up in #1441. The format is now a list of dicts as probably required for angular-data.

I dont' use the serializer because manual serializing the config data was pretty easy. For later update method we will have to implement a custom function in any case.